### ISSUE TITLE
Define index/gallery parent

### ISSUE THREAD

**Issue by chasebank** [dimsemenov/PhotoSwipe#1030]
The design/layout of my gallery requires that some images be wrapped in a second container div, which is causing them to appear as separate galleries

Example:

    div.gallery
        div.wrap
            images
        div.wrap
            images

PhotoSwipe finds the image parent, which is the wrap div, and makes that the galley. I'd like to specify the parent to avoid this, so that a user can swipe through all the images as one gallery. It looks like this addresses that:

    // find index of clicked item by looping through all child nodes
    // alternatively, you may define index via data- attribute

But it's not clear to me exactly what I should do.

**Comment by Saeglopur** [comment]
@chasebank You're trying to achieve something like that post in upper left corner? I'm struggling with that now! Example: http://transmissionclothing.tumblr.com/

**Comment by chasebank** [comment]
@Saeglopur Yes that's exactly it. That example has one large image in a div, and the three smaller images below are in a second div, but when you click on one, you're able to scroll through all four.

It looks like they're not using PhotoSwipe though. But I did notice they have `data-photoset-index="1"` on each link, cycling through 1, 2, 3 and 4. I suspect something similar is needed for PhotoSwipe to work with that layout, but I still can't figure it out.

Here's a reduced test case using that layout http://codepen.io/chasebank/pen/pgoMKr

**Comment by Saeglopur** [comment]
Thank you for reply @chasebank ! I noticed it just today. I went bit around that problem and I used FlatFolio plugin for Wordpress, it has nice slideshow but awful lot of other problems so I'm thinking now about replacing the lightbox. Here it is: http://workbench.pl/custom/ 

So you eventually used that what presented on Codepen? In my case it would be problematic to use it with current plugin I'm afraid.
